### ISSUE TITLE
Unable to edit failed sent e-mail

### ISSUE THREAD

**Issue by Almtesh** [thunderbird/thunderbird-android#2589]
### Expected behavior
When sending a e-mail has failed because of mistyping the recipient's address, edit the e-mail to correct the address, or move it to draft to edit it in the draft. The second option should abort to send e-mail.

### Actual behavior
Once in the outbox, an e-mail can only be sent again or deleted.

### Steps to reproduce
1. Write a new e-mail with a non-existent e-mail address as recipient
2. Send it!
3. Go to the outbox to try to move it to draft or edit it on place

### Environment
K-9 Mail version: 5.206

Android version: 6

Account type (IMAP, POP3, WebDAV/Exchange): IMAP/SMTP

Thanks.

**Comment by Prabashi** [comment]
I tried to reproduce the error and when the step 1 and step 2 are done, the e-mail is moved to Sent folder though the mail is not sent. The message can be forwarded and when the Forward is clicked, it moves to an interface where changes can be done to that mail.
K-9 mail version: 5.207
Android version: 4.2.2
Haven't I reproduced the issue correctly or the issue is addressed by the newer version of K9 mail?
Thanks.
